e2e-polybft/e2e/consensus_test.go

```diff
@@ -20,14 +20,43 @@ import (
 )
 
 func TestE2E_Consensus_Basic_WithNonValidators(t *testing.T) {
+	const epochSize = 4
+
+	//nolint:godox
```

Comment:
Did you want to skip this one as well?

Reply:
Nope, it is good to have these tests running. It is just going to be further updated via https://polygon.atlassian.net/browse/EVM-547, to add at least one more sub-test, which would test that non-validator node manages to sync up with the rest of the network. However these existing tests as they are right now, make sense per se. :slightly_smiling_face: